### ISSUE TITLE
Adding new features (OpenGraph, TwitterCard, ...)

### ISSUE THREAD

**Issue by Zodiac1978** [ArthurGareginyan/all-meta-tags#7]
Do you have any plans or interest in adding more features, like OpenGraph or Twitter Card?

If the plugin should stay small it is perfectly fine. Just asking to see if a PR could make sense.

**Comment by ArthurGareginyan** [comment]
Yes, we have interest in adding more features such as OpenGraph and Twitter Card.

We are already working on a major update of our plugin to make our users happy :) So any PRs would be very much appreciated. Thank you in advance!
